src/com/sun/ts/tests/jws/webmethod/webmethod1/server/standalone.web.xml

```diff
@@ -18,18 +18,20 @@
 -->
 
 <web-app version="5.0" xmlns="https://jakarta.ee/xml/ns/jakartaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd">
-  <display-name>@APPNAME@</display-name>
-  <servlet>
-    <servlet-name>@SERVLETNAME@</servlet-name>
-    <servlet-class>@IMPL@</servlet-class>
-    <load-on-startup>0</load-on-startup>
-  </servlet>
-  <servlet-mapping>
-    <servlet-name>@SERVLETNAME@</servlet-name>
-    <url-pattern>@URLALIAS@</url-pattern>
-  </servlet-mapping>
-  <session-config>
+  <display-name>DefaultWebMethodWebServiceApp</display-name>
+  <listener>
+<listener-class>com.sun.xml.ws.transport.http.servlet.WSServletContextListener</listener-class>
```

Comment:
Why are we now changing the various @SERVLETNAME@ (and other) parameters?

Reply:
Servlet containers are not required to implement enterprise ws-like deployment (aka jsr-109), see how other tests are written, ie at https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/src/com/sun/ts/tests/jaxws/ee/j2w/document/literal/webmethod/server

Reply:
Thanks Lukas!